src/test/resources/features/register/register.feature

```diff
@@ -3,16 +3,16 @@ Feature: Register user
   Scenario: Register user correctly
     Given User is on home page
     When he types unique e-mail
-    And on second registeration page he submits fields with correct data
+    And on second registration page he submits fields with correct data
     Then new user is registered
 
   Scenario: User tries to register by typing incorrect e-mail
     Given User is on home page
     When he submits form with incorrect e-mail
     Then error message is displayed
 
-  Scenario: User ries to register by typing incorrect data
+  Scenario: User tries to register by typing incorrect data
     Given User is on home page
     When he types new correct e-mail
     And incorrectly submits form on second registration page
-    Then error message is displayed
\ No newline at end of file
+    Then error message is displayed
```

Comment:
Zmień Given i wyrzuc do backgroundu:

  Background: 
    Given User is on login page
    
  Scenario: User is able to register in shop
    When he submits registration form with unique email
    And on second registration page he submits fields with correct data
    Then new user is registered